### DISCUSSION TITLE
[ 2021년 06월 08일 (화) ] 피어세션

### DISCUSSION THREAD

**Discussion by sglee487** [bcaitech1/p4-mod-model_diet#33] [General]
* lr을 0.1로 바꾸고, augmentation도 바꾸고 loss function을 custom으로 바꾸니까 train f1은 0.6이 나오는데 val f1은 0.49가 나옴.
* warmup 을 2epoch, step을 1500 정도.
* colab P100에선 batch_size 128이 1epoch당 6분 8초, batch_size 256이 1epoch 6분 20초 정도 나오고, batch_size 512가 7분 30초. 256이 제일 나은듯.
* 모델을 만들었다기보단 mobilenetv3 large -> small로 갈 때 어느 부분을 줄였나 보고 판단해서 임의로 줄였다.

## 진행했던 것
* 상건: automl로 MACs 5백만 찾아 학습중.
* 용우님: 모델 찾으려고 optuna 적용해보고 있다.
* 웅희님: MACs를 7백만, 5백만, 1백만 짜리로 만들어 학습 중.
* 연세님: shufflenetv2 0.5 돌려보려고 함. 또 용우님이 올려주신 좋은 모델도 돌려서 decomposition 해보겠다.
* 황훈님: optuna 아직 잘 안된다.
* 석진님: 포트폴리오 작성, tensor decomposition 실습파일 보고 있다.

## 앞으로 진행할거
* 상건: 모델 개조해보기, tensor decomposition 해보기.
* 석진님: tensor decomposition 실습파일 계속 보겠다.
* 웅희님: MACs이 작은 모델을 계속 만드는 작업을 하겠다.
* 황훈님: optuna 실습 하려고 함.
* 용우님: 모델 크기 계속 줄이는 방향으로. automl을 사용하면 좋을 것 같다.
* 연세님: 용우님이 올려주신 파일가지고 decomposition 해보기.
